values.yaml

```diff
@@ -85,6 +85,7 @@ injector:
     # Default values within Agent's template_config stanza.
     templateConfig:
       exitOnRetryFailure: true
+      staticSecretRenderInterval: "5m"
```

Comment:
I think it might be good to default this to empty `""`, and only add it to the deployment if it's been set by the user. That way it's not overriding the default in Vault Agent.

(The `exitOnRetryFailure` case was a little different since we explicitly wanted a different default setting for Kubernetes vs non-Kubernetes).